<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogToolClass</class>
 <widget class="QMainWindow" name="LogToolClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1241</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LogTool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="margin">
     <number>6</number>
    </property>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>4</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>850</width>
         <height>0</height>
        </size>
       </property>
       <property name="tabsClosable">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1241</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="openFileAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>窗口(&amp;I)</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>样式</string>
     </property>
     <addaction name="actionWindows"/>
     <addaction name="actionWindowsXP"/>
     <addaction name="actionWindowsVista"/>
     <addaction name="actionMotif"/>
     <addaction name="actionCDE"/>
     <addaction name="actionPlastique"/>
     <addaction name="actionCleanlooks"/>
    </widget>
    <addaction name="menu_4"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="openFileAction">
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="text">
    <string>存储</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>退出(&amp;E)</string>
   </property>
  </action>
  <action name="actionWindows">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Windows</string>
   </property>
  </action>
  <action name="actionWindowsXP">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>WindowsXP</string>
   </property>
  </action>
  <action name="actionWindowsVista">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>WindowsVista</string>
   </property>
  </action>
  <action name="actionMotif">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Motif</string>
   </property>
  </action>
  <action name="actionCDE">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>CDE</string>
   </property>
  </action>
  <action name="actionPlastique">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Plastique</string>
   </property>
  </action>
  <action name="actionCleanlooks">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Cleanlooks</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于(&amp;A)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="logtool.qrc"/>
 </resources>
 <connections/>
</ui>
